src/main/scala/beam/sim/BeamSim.scala

```diff
@@ -90,6 +91,11 @@ class BeamSim @Inject()(
         }
     }
 
+    beamServices.metricsPrinter = actorSystem.actorOf(MetricsPrinter.props())
+    Await.result(beamServices.metricsPrinter ? Identify(0), timeout.duration)
```

Comment:
Can we put this into a Future, then map to subscribe? We could then put off the blocking until the very last second. Or can everything else not currently handle the printer not being set? We should create an issue if so.